src/pages/listing/index.js

```diff
@@ -0,0 +1,533 @@
+                                    },
+                                    "\< Prev Page"
+                                ),
+                                m("span", {
+                                        class : css.currPage
+                                    },
+                                    isSearchResults ? "-" : ctrl.pg.page
+                                ),
+                                m("button", {
+                                        onclick  : ctrl.nextPage.bind(ctrl),
+                                        class    : css.nextPage,
+                                        disabled : locked || isSearchResults || ctrl.pg.page === ctrl.pg.numPages() || null
+                                    },
+                                    "Next Page \>"
+                                )
+                            ]);
+                        }())
+                    ),
+                    m("div", { class : css.listContainer }, 
+                        m("ul", { class : css.list },
+                            [
+                                m("li", { class : css.listHeader }, [
+                                    m("div", { class : css.listCol1 }, "Name"),
+                                    m("div", { class : css.listCol2 }, "State"),
+                                    m("div", { class : css.listCol3 }, "Updated"),
+                                    m("div", { class : css.listCol4 }, "Scheduled"),
+                                    m("div", { class : css.listCol5 }, "Actions")
+                                ])
+                            ].concat(
+                                content
+                                .sort(function(a, b) {
+                                    var aTime = a.order_by,
+                                        bTime = b.order_by;
+
+                                    // return aTime - bTime;
+                                    return bTime - aTime;
+                                })
+                                .map(function(data) {
+                                    var url      = "/content/" + ctrl.schema.key + "/" + data.key,
+                                        cssClasses = css.item,
+
+                                        itemName,
+                                        itemStatus,
+                                        itemUpdated,
+                                        itemSchedule;
+
+                                    if(data.published_at && current.indexOf(url) === 0) {
+                                        cssClasses = css.activePublished;
+                                    } else if(current.indexOf(url) === 0) {
+                                        cssClasses = css.active;
+                                    } else if(data.published_at) {
+                                        cssClasses = css.published_at;
+                                    }
+
+                                    if(isPast(data.unpublished_at)) {
```

Comment:
This should probably live in a utility method somewhere else because it (or code very similar to it) shows up a couple times.